src/components/transactions/transactionsFormatters.test.ts

```diff
@@ -31,16 +31,16 @@ describe("Given a call to format Transaction Number function", () => {
 });
 
 describe("Given a call to the format Transaction Price function", () => {
-    const amount = 50;
+    const amount = "50";

```

Comment:
Include the pennies to make this more realistic data
```suggestion
    const amount = "50.00";

```